source/crud/tests/v1/write/bulkWrite-collation.json

```diff
@@ -22,6 +22,7 @@
     }
   ],
   "minServerVersion": "3.4",
+  "serverless": "forbid",
```

Comment:
I missed this test because it passed in C# driver.
The `bulkWrite` operation didn't fail despite having a `collation` present. The difference with other operations is that they had `collation` specified as operation option while `bulkWrite` has this option set for the individual requests.
Which means that it is either an intended behavior or a bug in serverless implementation.

Reply:
To add another data point, these tests were [failing](https://evergreen.mongodb.com/task_log_raw/mongo_node_driver_ubuntu1804_test_serverless_test_serverless_patch_fa4352d297e70b83e48c1cb225eaab7cbe7404ab_60f07dbf0ae6061fe3feb9a7_21_07_15_18_26_13/0?type=T) in Node.